less/variables.less

```diff
@@ -24,3 +24,41 @@
 @border-real: #dee2e6;
 
 // Mobile var
+@phone: ~"(max-width: 500px)";
+
+// other mixins created
+.largestFont(){
+   font-size: 3.5rem; 
+//    width: 100%;
+}
+
+.anchorPropsNav(){
+    // text-decoration: none;
+    // margin:0% -65% 0 90%;
+    // font-size: 12px;
+    // color: black;
+    // padding: 100% 0%;
+    //  width: 100%;
+     text-decoration: none;
+     font-size: 12px;
+     color: black;
+     text-align: center;
+}
+
+.buttonTag{
+    background-color: black;
+    color: white;
+    cursor: pointer;
+    width: 90px;
+    height: 32px;
+    font-size: 1.5rem;
+    border-radius: 4px;
+ }
+ 
+// parametric mixin
+ .custom-tag(@justified, @aligned){
+	display: flex;
+	justify-content: @justified;
+	align-items: @aligned;
+}
+.custom-ltag(center, flex-start);
```

Comment:
It looks like you are tying to calling the `custom-tag` mixin here but the mixin is misspelled, it is not inisde of a selector(So it has no target to apply the mixin too). Keep in mind we write the mixins, and variables to separate files and import them to be used else where in the code (like home-page.less). This way we can effectively separate our concerns and make our code easier to work with.